pkg/resourcegroup/controllers/root/root_controller.go

```diff
@@ -296,3 +299,24 @@ func isConditionTrue(cType v1alpha1.ConditionType, conditions []v1alpha1.Conditi
 
 	return false
 }
+
+type OwnedByConfigSyncPredicate struct{}
+
+func (OwnedByConfigSyncPredicate) Create(e event.CreateEvent) bool {
+	return isOwnedByConfigSync(e.Object)
+}
+func (OwnedByConfigSyncPredicate) Delete(e event.DeleteEvent) bool {
+	return isOwnedByConfigSync(e.Object)
+}
+func (OwnedByConfigSyncPredicate) Update(e event.UpdateEvent) bool {
+	return isOwnedByConfigSync(e.ObjectOld)
```

Comment:
We would also need to check if the new object is owned by Config Sync.
`return isOwnedByConfigSync(e.ObjectOld) || isOwnedByConfigSync(e.ObjectNew)`

Reply:
Thanks! I was unsure about what the best policy was here; fixed in 60bde30.